partner_credit_limit/migrations/11.0.0.0.2/post-migration.py

```diff
@@ -0,0 +1,28 @@
+from odoo import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    set_credit_limit_values_to_partner(cr)
+
+
+def set_credit_limit_values_to_partner(cr):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    env.cr.execute(
+        """
```

Comment:
My last review was about using api instead of query here

Reply:
@moylop260 For that @hugho-ad replied this:

> because this is a post-migration script, so after the field change values returned by the api like
> `partner.credit_limit` is False,
> that why the migration script in order to recover that values from db


Regards.

Reply:
You need using domain with None or empty string

Reply:
Hi @moylop260 I was trying to test your advice. 

This is my tests:
1. Install the module before the `company_dependent` change.
2. Set values to fields `credit_limit` and `grace_payment_days` for 6 partners.
3. Update the module with the `company_dependent` change, without the migration.
4. Set values to fields `credit_limit` and `grace_payment_days` for 2 more partners.
5. Update the module with the migration and this is the result:
     When I execute the sql query, the result is 6 partners.
     When I execute the search the result is 2 partners.
     When I execute the search with None or ' ' I get an error.

I left this image with a pdb in the migration script.

<img width="1428" alt="Screen Shot 2019-10-25 at 12 10 42 PM" src="https://user-images.githubusercontent.com/11479473/67590928-c73a9f00-f721-11e9-847f-3e10d804275c.png">


Please let me know your comments. 

Thanks  in advance.

cc @hugho-ad

Reply:
Cool!
I understand that after our meeting Edi.

The matter here is that the api is adapted for new definition of the field using ir.property (multi-company field).
So, it is not possible get the old values since that the search will be for ir.property table instead of res.partner like old way.

So, let's using query directly.

Reply:
Yes! Maybe my explanations was not totally clear! Thanks for your time! @moylop260  😄